### ISSUE TITLE
Change tooltip position based on toolbar position

### ISSUE THREAD

**Issue by kankaristo** [suda/tool-bar#68]
Thanks for a great package, makes Atom feel a lot more like an IDE. :)

However, the tooltips for the buttons appear on top of the buttons (I have my toolbar on the left side).

[`TooltipManager::add`](https://atom.io/docs/api/v0.207.0/TooltipManager#instance-add) has an option to set the position (placement) of the tooltip.

The tooltip position should ideally be opposite of the toolbar position. So if the toolbar is at the top, the tooltip should be shown below; if the toolbar is on the left, the tooltip should be on the right.

This is what I did in `tool-bar-button-view.coffee`:
```coffee
if options.tooltip
  @prop 'title', options.tooltip
  @subscriptions.add atom.tooltips.add(
    this,
    {
      title: options.tooltip
      delay:
        show: 0
        hide: 0
      placement: 'right'  # Opposite of the toolbar's position
    }
  )
```

(I also set the delay to 0, by personal preference).

Do you think this could be made the default behavior?

**Comment by jerone** [comment]
Hi @kankaristo, thanks for using this package.

I like your idea of placing the tooltips to the side when the tool-bar is vertical. On a horizontal tool-bar the tooltips are already placed correctly.

For the `delay` part, I prefer to keep this in sync with the [tabs package](https://github.com/atom/tabs/blob/18b1a46a23cd460a8f1562ddb4a3ed17c0594f50/lib/tab-view.coffee#L96-L98) or default value as with [status-bar package](https://github.com/atom/status-bar/search?utf8=%E2%9C%93&q=tooltip).

Would you like to do a PR?

**Comment by kankaristo** [comment]
Pull request: https://github.com/suda/tool-bar/pull/69

**Comment by kankaristo** [comment]
Fixed in b2a9e65b8825e0c598726fd767d9fae5434205ac
